src/__tests__/screenTestRunner.py

```diff
@@ -29,19 +32,21 @@ def getLineObjsFromFile(inputFile):
 
 
 def getRowsFromScreenRun(
-      inputFile,
-      charInputs,
-      screenConfig={},
-      printScreen=True,
-      pastScreen=None):
+        inputFile,
+        charInputs,
+        screenConfig={},
+        printScreen=True,
+        pastScreen=None):
     lineObjs = getLineObjsFromFile(inputFile)
     screen = ScreenForTest(
         charInputs,
         maxX=screenConfig.get('maxX', 80),
         maxY=screenConfig.get('maxY', 30),
     )
+    # mock our flags with an empty command line for now
+    flags = ScreenFlags.initFromArgs([])
```

Comment:
ah i didnt realize it was so easy to mock arg-based parsing... aka i didn't realize `argparse` could take in an empty array and not complain